index.js

```diff
@@ -49,8 +49,7 @@ module.exports = {
 				'unicorn/no-useless-undefined': 'error',
 				'unicorn/no-zero-fractions': 'error',
 				'unicorn/number-literal-case': 'error',
-				// TODO: Enable this by default when it's shipping in a Node.js LTS version.
-				'unicorn/numeric-separators-style': 'off',
```

Comment:
The comment was wrong. We need to wait until April next year when Node.js 10 is obsolete. https://github.com/nodejs/Release

Reply:
Oh ok 🙂